JS101/final-project.md

```diff
@@ -3,3 +3,10 @@ Ideas for final JS101 project:
 tictactoe => use elium stuff.
 on screen calculator?
 
+
+
+https://github.com/jankeLearning/projects/tree/master/04-be-you-tube
```

Comment:
this is the end of the basics of the JS section... so making youtube is probably not appropriate just yet... would be a good project for later in the course though.